### DISCUSSION TITLE
Leaderboard card

### DISCUSSION THREAD

**Discussion by Beaueyjangles** [neplexlabs/canvacord#167] [Ideas]
Hello, 

i have noticed with most bots now using canvacord for there player rank card. If someone with the skills could make a leaderboard in the same style as the playercard. that would be awesome.

**Comment by twlite** [comment]
please use canvacord v6 for this feature

https://canvacord.js.org/guide/examples/leaderboard-image-generation.md

![image](https://github.com/neplextech/canvacord/assets/46562212/508147c1-d48f-4d77-9058-ce774cb5899c)
